internal/server/data/data_test.go

```diff
@@ -138,44 +138,51 @@ func TestDatabaseSelectors(t *testing.T) {
 
 func TestPaginationSelector(t *testing.T) {
 	runDBTests(t, func(t *testing.T, db *gorm.DB) {
-		letters := make([]string, 0, 26)
+		alphabeticalIdentities := []string{}
 		for r := 'a'; r < 'a'+26; r++ {
-			letters = append(letters, string(r))
+			alphabeticalIdentities = append(alphabeticalIdentities, string(r))
 			g := &models.Identity{Name: string(r)}
 			assert.NilError(t, CreateIdentity(db, g))
 		}
 
+		alphabeticalIdentities = append(alphabeticalIdentities, models.InternalInfraConnectorIdentityName)
```

Comment:
Why add the connector identity? Seems like a lot of trouble for just seeing it paginate correctly.

Reply:
Good call-out, it made more sense to remove the connector identity from the returned list of all identities. Made that change.